buidl/script.py

```diff
@@ -428,3 +428,22 @@ def p2sh_address(self, testnet=False):
         redeem_script = self.script_pubkey().redeem_script()
         # return the p2sh address of the RedeemScript (remember testnet)
         return redeem_script.address(testnet)
+
+    def is_p2wsh(self):
+        return (
+            OP_CODE_NAMES[self.commands[-1]] == "OP_CHECKMULTISIG"
+            and type(self.commands[0]) == int
+            and type(self.commands[-2]) == int
+        )
+
+    def get_quorum(self):
+        """
+        Return the m-of-n of this multisig, as in 2-of-3 or 3-of-5
+        """
+
+        assert self.is_p2wsh(), f"Not a multisig witness script: {self}"
```

Comment:
I would prefer a specific Exception not a bald assert.

Reply:
I changed it to this locally:
```
        if not self.is_p2wsh_multisig():
            raise RuntimeError(f"Not a multisig witness script: {self}")
```

Is that what you had in mind? I could define an Exception class just for catching this error, but I didn't because I thought it too much clutter.

Reply:
Maybe ValueError or SyntaxError? Those are built in and seem more appropriate here.